src/qt/rpcconsole.cpp

```diff
@@ -25,7 +25,7 @@
 
 #ifdef ENABLE_WALLET
 #include <db_cxx.h>
-#include <wallet/wallet.h>
+#include "wallet/wallet.h"
```

Comment:
Same comment here. Making this a relative include makes the compiler try `src/qt/wallet/wallet.h`, which doesn't exist.